docs/UserGuide.md

```diff
@@ -317,10 +346,19 @@ Format: `status INDEX s/STATUS`
 * Edits the applicant at the specified `INDEX`. The index refers to the index number shown in the displayed applicant list.
 * `INDEX` must be a positive integer (e.g. `1`, `2`, `3`, ...).
 
-Example:
-* `status 2 s/r` updates the status of the 2nd person in the displayed applicant list to _REJECTED_.
+Valid examples:
+* `status 3 s/o` updates the status of the 3rd person in the displayed applicant list to _OFFERED_.
+* `status 1 s/r` updates the status of the 1st person in the displayed applicant list to _REJECTED_.
+
+Invalid examples:
+* `status 3 s/interviewing`
+This is not allowed as `STATUS` must be either `o(ffered)` or `r(ejected)` or `u(ndecided)`.
 
-![status](images/user-guide/status.png)
+* `status 3`
+This is not allowed as a `STATUS` must be provided.
+
+* `status -3 s/o`
+This is not allowed as an `INDEX` must be a positive integer.
```

Comment:
same thing here too!

Reply:
Hi ivan thanks for the review. I've changed it accordingly. Does it look better now?